beacon-chain/sync/subscriber.go

```diff
@@ -240,6 +236,7 @@ func (s *Service) registerSubscribers(nse params.NetworkScheduleEntry) bool {
 		s.spawn(func() {
 			s.subscribe(
 				p2p.SyncContributionAndProofSubnetTopicFormat,
+				s.buildTopicWithoutSubnet(p2p.SyncContributionAndProofSubnetTopicFormat, nse.ForkDigest),
```

Comment:
- All the parameters used in `s.buildTopicWihoutSubnet` are also parameters of `subscribe`.
- All calls of `s.subscribe` also use `s.builTopicWithoutSubnet`

==> `s.buildTopicWithoutSubnet` should be directly called in `s.subscribe`.